src/DependencyInjection/Configuration.php

```diff
@@ -891,7 +903,40 @@ private function addMongoSection(ArrayNodeDefinition $handlerNode)
             ->end()
             ->validate()
                 ->ifTrue(function ($v) { return 'mongo' === $v['type'] && !isset($v['mongo']); })
-                ->thenInvalid('The mongo configuration has to be specified to use a MongoHandler')
+                ->thenInvalid('The "mongo" configuration has to be specified to use a "mongo" handler type.')
+            ->end()
+        ;
+    }
+
+    private function addMongoDBSection(ArrayNodeDefinition $handlerNode)
+    {
+        $handlerNode
+            ->children()
+                ->arrayNode('mongodb')
+                    ->canBeUnset()
+                    ->beforeNormalization()
+                        ->ifString()
+                        ->then(function ($v) { return ['id' => $v]; })
+                    ->end()
+                    ->children()
+                        ->scalarNode('id')->end()
```

Comment:
We can document the configuration:
```suggestion
                        ->scalarNode('id')
                            ->info('ID of a MongoDB\Client service')
                            ->example('doctrine_mongodb.odm.logs_connection')
                        ->end()
```

Reply:
Ah, I love that we can add inline documentation like this. That will be super useful for [PHPLIB-1741](https://jira.mongodb.org/browse/PHPLIB-1741).